City-of-Helsinki/drupal-module-helfi-api-base
# UHF-2504 partial migrate

Adds an event lister to resolve whether the next migration should be full migration or partial migration.

Test:
- Select one of the websites which uses the api_base module 
- `composer require drupal/helfi_api_base:dev-UHF-2504_partial_migrate`
- `Make new`
- Run migrations manually `MIGRATE_LIMIT=10 drush mim tpr_unit`
- key_value table should now contain the migration state data: `drush sqlc` and `select * from key_value where name like 'partial_migrate%'`
